setup.py

```diff
@@ -84,7 +84,6 @@ def run_tests(self):
         'Natural Language :: English',
         'License :: OSI Approved :: Apache Software License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2.6',
```

Comment:
Added `'Programming Language :: Python :: 2'` Keep in mind that `2.7` is still there as well.